zanata-war/src/main/java/org/zanata/rest/service/ResourceUtils.java

```diff
@@ -108,6 +109,7 @@
     private static final String CONTENT_TYPE_HDR = HeaderFields.KEY_ContentType;
     private static final String PLURAL_FORMS_HDR = "Plural-Forms";
 
+    // TODO we should need only one regex, with a capture group for nplurals
```

Comment:
There are three patterns in this file, but they didn't change in this PR, so they're not entirely visible here.